### ISSUE TITLE
ui5-list inside ui5-popover issue when using JAWS or NVDA

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#3053]
**Describe the bug**
This is a strange bug, it is only reproducible while a screen reader like JAWS or NVDA (not macOS VoiceOver) is running. When a ui5-list is inside a ui5-popover and the list items have role="button", the keyboard navigation with arrow keys stops working. 

**To reproduce**
1. Use JAWS or NVDA.
2. Go to https://codesandbox.io/s/ui5-webcomponents-forked-jqx6h
3. Follow instructions at the top of the sandbox
4. Notice the keyboard navigation with arrow keys is not working for list items that have role="button"

**Expected behavior**
The goal is to use the ui5-popover and ui5-list to make a menu with actions that can be clicked or operated by keyboard. The keyboard navigation should not be broken by having a screen reader running.

**Context**
- UI5 Web Components version 0.29.9
- OS/Platform: Windows 10

**Affected components** 
ui5-popover
ui5-list
ui5-li

**Organization:**
SuccessFactors

**Priority:** *(optional)* 
Medium

Thank you for looking into this strange issue. Hope you are able to reproduce it.

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Keyboard navigation doesn't work in the list, when screen reader tool is started.
The issue is reproducible with the latest release on windows 10 machine, chrome browser and JAWS 2019 and with
the provided by the author steps, which indicate what is the real use case.

The issue could also be reproduced by using only a ui5-list component via the following steps:
1) Open the following snipped:
https://codesandbox.io/s/ui5-webcomponents-forked-dg52n?file=/index.html
2) Start JAWS tool
3) Navigate to the list item with text "one" of the first list
4) Press arrow down keyboard key

Expected:
Focus to be moved to the list item with text "two".

Current:
Keyboard navigation doesn't work in the list.

In comparison the issue isn't reproducible with the second list, where there is no role="button" attributes added to the list items.

Every ui5-li element has "actionable" attribute, which if removed the keyboard navigation would work.

Could you please take over.

Best Regards,
Boyan

**Comment by ee92** [comment]
@dobrinyonkov Hi, sorry for the slow reply. I'm not sure that will be a good solution for us, because customers will expect something like tabbing between buttons to work without using a special screen reader mode. Is there any plan to fix this a different way? Thanks

**Comment by dobrinyonkov** [comment]
Hello @ee92, apologies for the inconvenience, the issue actually is that you are setting the `role="button"` to the listitems which breaks the accessiblity tree mapping, since according to WAI-ARIA the only valid children of a `list` should be `listitem` elements or role `group` element with `listitem` inside.

Also, the `role` property of the `ui5-list` component is private and should not be used.

**Comment by dobrinyonkov** [comment]
Hello @SAP/ui5-webcomponents-topic-core, this is a feature request for a Menu component.

Kind Regards,
Dobrin

**Comment by ilhan007** [comment]
Hello @ee92 
The required behaviour is provided by the Menu web component https://sap.github.io/ui5-webcomponents/master/playground/components/Menu/
